{
    "type": "bundle",
    "id": "bundle--b984999b-2b17-42a7-8565-f37ec2eeb133",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--80cdae98-f296-4501-ae13-ad585554dbdd",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-12-13T00:21:30.834677Z",
            "modified": "2024-12-13T00:21:30.834677Z",
            "name": "CVE-2024-54513",
            "description": "A permissions issue was addressed with additional restrictions. This issue is fixed in watchOS 11.2, visionOS 2.2, tvOS 18.2, macOS Sequoia 15.2, iOS 18.2 and iPadOS 18.2. An app may be able to access sensitive user data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-54513"
                }
            ]
        }
    ]
}